### DISCUSSION TITLE
[Performance] : Maximum Load Capacity Of Skywalking.

### DISCUSSION THREAD

**Discussion by the-rough-page** [apache/skywalking#8765] [Q&A] [answered]
Can Someone Please Explain, How much loads this skywalking Collector Can Take For a Single Application? I mean to say like if we are monitoring an application which is having a lot of transactions and processes then is there any limitation for Skywalking to accommodate and store the whole and monitor the same.

**Comment by wu-sheng** [comment, marked as answer]
This is pointless. If one OAP could support more than one application, you mean you have a larger cluster for monitoring than target services.
